extensions/amp-sidebar/1.0/storybook/Basic.amp.js

```diff
@@ -31,10 +31,45 @@ export default {
 
 export const _default = () => {
   const sideConfigurations = ['left', 'right', undefined];
-  const side = select('type', sideConfigurations, sideConfigurations[0]);
+  const side = select('side', sideConfigurations, sideConfigurations[0]);
+  const foregroundColor = color('color');
+  const backgroundColor = color('background');
+  const height = text('height', null);
+  const width = text('width', null);
+  const padding = text('padding', null);
+  const border = text('border', null);
+  const top = text('top', null);
+  const maxHeight = text('max-height', null);
+  const maxWidth = text('max-width', null);
+  const minWidth = text('min-width', null);
+  const outline = text('outline', null);
+  const zIndex = number('z-index', 2147483647);
+  const backdropColor = color('backdrop color');
```

Comment:
This many configurations feels clunky and can obscure other functional knobs like the sides. What do you think about adding a custom styles sample that is separate from default?

Reply:
Great idea.  I've added an additional story for `styles`.  In the `default` story I've kept `color`, `background`, and `backdrop color`, so the user can still get a taste of configuring the sidebar w/o getting overwhelmed.